core/deployment/pom.xml

```diff
@@ -187,6 +187,8 @@
             <plugin>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
+                    <source>10</source>
+                    <target>10</target>
```

Comment:
Stupid question but why `10`?

Reply:
I had the same question myself, but didn't bother to ask because that's already what we had. But we should definitely make sure it's what we want and if not, change it in a separate commit

Reply:
Don't ask https://github.com/quarkusio/quarkus/commit/b16b99250927da55069675701ba243b896ea7f6c

Reply:
Well, actually, I'm asking. I don't understand why we would enforce a separate source/target here. Especially using version `10`.

@patriot1burke could you comment on why you did that? Because I would prefer to see it gone.

Reply:
(Note that it's not blocking this PR, I just want this sorted out and I will adjust in a subsequent PR if need be)

Reply:
I'm guessing this is a mistake...Like I think Intellij may have inserted it when me (or Clement?  probably me) set the language level.

Reply:
OK, I will remove it then.